app/docs/dev/networking/transparent-proxying.md

```diff
@@ -202,3 +201,41 @@ networking:
 ```
 {% endtab %}
 {% endtabs %}
+
+### Transparent Proxy using eBPF (experimental)
+
+Starting from {{site.mesh_product_name}} 2.0 you can setup transparent proxy to use eBPF instead of iptables.
+
+{% warning %}
+To use Transparent Proxy with eBPF your environment has to use `Kernel >= 5.7`
+and have `cgroup2` available
+{% endwarning %}
+
+{% tabs ebpf useUrlFragment=false %}
+{% tab ebpf Kubernetes %}
+
+```shell
+kumactl install control-plane \
+  --set "kuma.experimental.ebpf.enabled=true" | kubectl apply -f-
+```
+
+{% endtab %}
+
+{% tab ebpf Universal %}
+
+```shell
+kumactl install transparent-proxy \
```

Comment:
Not sure if for universal it is going to work that way. I think you need provide path to ebpf programs

Reply:
you are right, updating the PR